test_conformance/api/test_api_min_max.cpp

```diff
@@ -1297,7 +1298,7 @@ int test_min_max_constant_buffer_size(cl_device_id deviceID, cl_context context,
     size_t    threads[1], localThreads[1];
     cl_int *constantData, *resultData;
     cl_ulong maxSize, stepSize, currentSize, maxGlobalSize, maxAllocSize;
-    int i;
+    cl_ulong i;
```

Comment:
Somewhat pedantic: should this be a `size_t` instead of a `cl_ulong`?  It seems to be used as a loop counter over `numberOfInts` elements, and `numberOfInts` is a `size_t`.

If we do switch to a `size_t` please check the printf formats specifiers also, to be sure we're properly using `%zu` vs. `%llu`.

Reply:
fixed